css/responsive/desktop.styl

```diff
@@ -0,0 +1,18 @@
+$cols = 12
+$gutter = 32px
+
+.container
+  width: 1150px
+  
+.col-l
+  padding-right: ($gutter / 2)
```

Comment:
the problem with having the left and right padding on the `.col` is that it resets the flex, which means we would need to have the repetitive `col-m-6 col-l-6` for example